src/wp-includes/rest-api/endpoints/class-wp-rest-font-families-controller.php

```diff
@@ -0,0 +1,554 @@
+		 */
+		return apply_filters( 'rest_wp_font_family_collection_params', $query_params );
+	}
+
+	/**
+	 * Get the arguments used when creating or updating a font family.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @return array Font family create/edit arguments.
+	 */
+	public function get_endpoint_args_for_item_schema( $method = WP_REST_Server::CREATABLE ) {
+		if ( WP_REST_Server::CREATABLE === $method || WP_REST_Server::EDITABLE === $method ) {
+			$properties = $this->get_item_schema()['properties'];
+			return array(
+				'theme_json_version'   => $properties['theme_json_version'],
+				// When creating or updating, font_family_settings is stringified JSON, to work with multipart/form-data.
+				// Font families don't currently support file uploads, but may accept preview files in the future.
+				'font_family_settings' => array(
+					'description'       => __( 'font-family declaration in theme.json format, encoded as a string.' ),
+					'type'              => 'string',
+					'required'          => true,
+					'validate_callback' => array( $this, 'validate_font_family_settings' ),
+					'sanitize_callback' => array( $this, 'sanitize_font_family_settings' ),
+				),
+			);
+		}
+
+		return parent::get_endpoint_args_for_item_schema( $method );
+	}
+
+	/**
+	 * Get the child font face post IDs.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param int $font_family_id Font family post ID.
+	 * @return int[] Array of child font face post IDs.
+	 */
+	protected function get_font_face_ids( $font_family_id ) {
+		$query = new WP_Query(
+			array(
+				'fields'                 => 'ids',
+				'post_parent'            => $font_family_id,
+				'post_type'              => 'wp_font_face',
+				'posts_per_page'         => 99,
+				'order'                  => 'ASC',
+				'orderby'                => 'id',
+				'update_post_meta_cache' => false,
+				'update_post_term_cache' => false,
+			)
+		);
+
+		return $query->get_posts();
+	}
+
+	/**
+	 * Prepares font family links for the request.
+	 *
+	 * @since 6.5.0
+	 *
+	 * @param WP_Post $post Post object.
+	 * @return array Links for the given post.
+	 */
+	protected function prepare_links( $post ) {
+		// Entity meta.
+		$links = parent::prepare_links( $post );
+
+		return array(
+			'self'       => $links['self'],
+			'collection' => $links['collection'],
+			'font_faces' => $this->prepare_font_face_links( $post->ID ),
+		);
+	}
+
+	/**
+	 * Prepares child font face links for the request.
+	 *
+	 * @param int $font_family_id Font family post ID.
+	 * @return array Links for the child font face posts.
+	 */
+	protected function prepare_font_face_links( $font_family_id ) {
+		$font_face_ids = $this->get_font_face_ids( $font_family_id );
+		$links         = array();
+		foreach ( $font_face_ids as $font_face_id ) {
+			$links[] = array(
+				'embeddable' => true,
+				'href'       => rest_url( $this->namespace . '/' . $this->rest_base . '/' . $font_family_id . '/font-faces/' . $font_face_id ),
```

Comment:
Nit: in other REST controllers we usually use `sprintf` for assembling REST URLs like this.

Reply:
I'm addressing this in this Gutenberg PR: https://github.com/WordPress/gutenberg/pull/58736. Once merged, I'll commit to this PR.